src/components/QueryRuleCustomData.vue

```diff
@@ -36,7 +36,7 @@ export default {
     transformItems: {
       type: Function,
       required: false,
```

Comment:
do we need to state it if it's `false`?

Reply:
It doesn't really make much difference, so required defaults to true